Library/PhotosController.swift

```diff
@@ -50,6 +50,20 @@ class PhotosController: UICollectionViewController {
         }
     }
 
+    #if os(tvOS)
+    override var preferredFocusEnvironments: [UIFocusEnvironment] {
```

Comment:
In this class the main changes are improving the control of the focus engine. `preferredFocusEnvironments` should return the views that you want to focus, in our case it will return the currently selected cell.